src/gsi/projmethod_support.f90

```diff
@@ -1,394 +1,3 @@
-module projmethod_support
```

Comment:
Is projmethod_support.f90 still an appropriate name for this file?

Reply:
Probably not.  Was trying to make minimal changes for this.  However could put in another file name or include in one of the other existing files.  Any suggestions?

Reply:
writeout_gradients is only called in pcgsoi, so I suggest that this subroutine could be moved to this module. Or that this file can just be renamed writeout_gradients.f90

Reply:
Good idea.  I am moving writeout_gradients to pcgsoi.f90 and removing projmethod_support.f90